### PR TITLE
Fix the path for the spinning logo ajax-loader.gif

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/15305 .

### Summary of Changes
Using data attributes for passing php data to javascript is totally fine (and very welcome) but in this particular occasion we need another way, since all the front end templates either need to update to echo this data attribute (not very likely to happen) or will not work.

So lets use the language strings (since what we're passing is just a word/string)...

### Testing Instructions

Create a com_content custom field and edit an article from the front end.

### Actual result

no errorSpinning logo appears correctly, no error 404 not found in the browser's console

### Documentation Changes Required
Nope

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 8745f48c9bc20971bf9f4f86c2d962d63b2bfd61<br /><br />something in this PR is killing things like patchtester<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15529">issues.joomla.org/tracker/joomla-cms/15529</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
Open an Article, Tab "Content" or Tab "Fields", got in Console/JS: Use of getAttributeNode() is deprecated. Use getAttribute() instead.  jquery-migrate.min.js:2:1714

File: libraries/cms/html/behavior.php [hunk 1]
```diff
@@ -87,6 +87,9 @@ public static function core()
 			return;
 		}
 
+		// Define the base path as a string
+		JText::script('JOOMLA_BASE_PATH', JURI::root(true));
```
Comment (Fedik) [inline libraries/cms/html/behavior.php]: that is sad 😄

Comment (Fedik) [inline libraries/cms/html/behavior.php]:
```php
JFactory::getDocument()
  ->addScriptOptions('core.root', JURI::root(true))
  ->addScriptOptions('core.base', JURI::base(true));
```
or `joomla.root`, whatever 😉

Comment (dgt41) [inline libraries/cms/html/behavior.php]: @Fedik sure, that looks more appropriate

**Comment by AlexRed** [comment]
I can confirm the problem on Joomla! 3.7.2-Dev (nightly Build)

**Comment by brianteeman** [comment]
after applying the patch all i get anywhere is

```
Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in C:\Apache24\htdocs\cms\libraries\cms\html\html.php on line 66

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in Unknown on line 0

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in Unknown on line 0

Warning: Unknown: Cannot call session save handler in a recursive manner in Unknown on line 0
```

**Comment by Fedik** [comment]
@dgt41 there an alternative solution #16701

**Pull request closed without merging**